test/fixtures/permission/fs-traversal.js

```diff
@@ -98,11 +98,6 @@ const uint8ArrayTraversalPath = new TextEncoder().encode(traversalPath);
     return w.apply(this, [traversalPath, ...args]);
   })(Buffer.prototype.utf8Write);
 
-  // Sanity check (remove if the internals of Buffer.from change):
-  // The custom implementation of utf8Write should cause Buffer.from() to encode
-  // traversalPath instead of the sanitized output of resolve().
-  assert.strictEqual(Buffer.from(resolve(traversalPathWithExtraChars)).toString(), traversalPath);
```

Comment:
If I understood correctly this can be deleted now that `Buffer.prototype.utf8Write` tampering doesn't impact from/toString. However, the test still has value (along with the tampering code above) if that were to be reversed at any point.